atomic_reactor/plugins/post_export_operator_manifests.py

```diff
@@ -0,0 +1,179 @@
+"""
+Copyright (c) 2019 Red Hat, Inc
+All rights reserved.
+
+This software may be modified and distributed under the terms
+of the BSD license. See the LICENSE file for details.
+"""
+
+import tarfile
+import json
+import shutil
+import os
+import tempfile
+
+from atomic_reactor.constants import PLUGIN_EXPORT_OPERATOR_MANIFESTS_KEY
+from atomic_reactor.plugin import PostBuildPlugin
+from atomic_reactor.util import df_parser, is_scratch_build, get_platforms
+
+OPERATORS_LABEL = 'com.redhat.delivery.appregistry'
+MANIFESTS_DIR_NAME = 'manifests'
+OPERATORS_ARCHIVE_NAME = 'operator_manifests'
+OPERATORS_ARCHIVE_EXT = 'zip'
+IMAGE_MANIFEST_NAME = 'manifest.json'
+WHITEOUT_FILE_PREFIX = '.wh.'
+
+
+class ExportOperatorManifestsPlugin(PostBuildPlugin):
+    """
+    Export operator manifest files
+
+    Fetch and archive operator manifest files from image so they can be
+    uploaded to koji.
+    """
+
+    key = PLUGIN_EXPORT_OPERATOR_MANIFESTS_KEY
+    is_allowed_to_fail = False
+
+    def __init__(self, tasker, workflow):
+        """
+        constructor
+
+        :param tasker: DockerTasker instance
+        :param workflow: DockerBuildWorkflow instance
+        """
+        super(ExportOperatorManifestsPlugin, self).__init__(tasker, workflow)
+
+    def has_operator_manifest(self):
+        """
+        Check if the plugin is running in orchestrator.
+
+        :return: bool
+        """
+        dockerfile = df_parser(self.workflow.builder.df_path, workflow=self.workflow)
+        # TODO: set the label in util Labels
```

Comment:
I'd say we want to add it to Labels in osbs-client's utils, as we are trying to keep all label constants in one place